src/AzureAuth/Commands/CommandAad.cs

```diff
@@ -27,34 +27,77 @@ namespace Microsoft.Authentication.AzureAuth.Commands
     {
-        private const string ResourceOption = "--resource";
-        private const string ClientOption = "--client";
-        private const string TenantOption = "--tenant";
-        private const string PromptHintOption = "--prompt-hint";
-        private const string ScopeOption = "--scope";
-        private const string ClearOption = "--clear";
-        private const string DomainOption = "--domain";
-        private const string ModeOption = "--mode";
-        private const string OutputOption = "--output";
-        private const string AliasOption = "--alias";
-        private const string ConfigOption = "--config";
-        private const string PromptHintPrefix = "AzureAuth";
-        private const string TimeoutOption = "--timeout";
+        /// <summary>
+        /// The Domain option.
+        /// </summary>
+        public const string DomainOption = "--domain";
+
+        /// <summary>
+        /// The Tenant option.
+        /// </summary>
+        public const string TenantOption = "--tenant";
+
+        /// <summary>
+        /// The Timeout option.
+        /// </summary>
+        public const string TimeoutOption = "--timeout";
+
+        /// <summary>
+        /// The Mode option.
+        /// </summary>
+        public const string ModeOption = "--mode";
+
+        /// <summary>
+        /// The Prompt Hint option.
+        /// </summary>
+        public const string PromptHintOption = "--prompt-hint";
+
+        /// <summary>
+        /// The Prompt Hint help text.
+        /// </summary>
+        public const string PromptHintHelpText = "A prompt hint to contextualize prompts and identify uses in telemetry";
 
 #if PlatformWindows
-        private const string AuthModeHelperText = @"Authentication mode. Default: iwa (Integrated Windows Auth), then broker, then web.
+        /// <summary>
+        /// The help text for the <see cref="ModeOption"/> option.
+        /// </summary>
+        public const string AuthModeHelperText = @"Authentication mode. Default: iwa (Integrated Windows Auth), then broker, then web.
 You can use any combination of modes with multiple instances of the --mode flag.
 Allowed values: [all, iwa, broker, web, devicecode]";
 #else
-        private const string AuthModeHelperText = @"Authentication mode. Default: web.
+        /// <summary>
+        /// The help text for the <see cref="ModeOption"/> option.
+        /// </summary>
+        public const string AuthModeHelperText = @"Authentication mode. Default: web.
 You can use any combination with multiple instances of the --mode flag.
 Allowed values: [all, web, devicecode]";
 #endif
 
+        /// <summary>
+        /// The help text for the <see cref="DomainOption"/> option.
+        /// </summary>
+        public const string DomainHelpText = "Preferred domain to filter cached accounts by.If a single account matching the preferred domain is in the cache it is used, otherwise an account picker will be launched.\n";

```

Comment:
```suggestion
        public const string DomainHelpText = "Preferred domain to filter cached accounts by. If a single account matching the preferred domain is in the cache it is used, otherwise an account picker will be launched.\n";
```